protos/in_toto_attestation/v1/resource_descriptor.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package io.in_toto.attestation.v1_0;
+package in_toto_attestation.v1;
 
 import "google/protobuf/struct.proto";
 
-option go_package = "github.com/in-toto/attestation/go/spec/v1.0";
+option go_package = "github.com/in-toto/attestation/go/v1";
```

Comment:
Why drop the `.0`?

We dropped the `.0` from the type string because any new fields are supposed to be backwards compatible unless or else they need to increment the major number.  However we may want to make backwards compatible changes to these messages, which would rev the minor version, even though the major version and type string would be unchanged.

So I think it's appropriate to keep `.0` here so that we can introduce `1.1` in the future with new fields.

Or maybe, the argument goes, if it really is backwards compatible, then producers of the v1 proto would be unaffected by new fields and could just transparently take the new definition.  Is that right?

Reply:
> if it really is backwards compatible, then producers of the v1 proto would be unaffected by new fields and could just transparently take the new definition. Is that right?

Yes, this is the intent. I do realize now that we probably want to document basic protos rules to facilitate this: 
* no field re-ordering within a major version
* no field type changes within a major version
* no required fields

Would this work? Also, is there a way to encode the current proto spec version into the definition?

Reply:
Yeah that makes sense.  Maybe we can just link to this https://protobuf.dev/programming-guides/proto3/#updating ?

> is there a way to encode the current proto spec version into the definition?

What do you mean by proto spec version?

Reply:
> What do you mean by proto spec version?

The statement spec is currently at version 1.0, so is there a way to encode/validate this version number in the corresponding proto? I suspect the answer is no.

Reply:
Oh I see.  That's something I had wanted to use cuelang.org and vet for.  Theoretically you could create cuelang definitions to validate all the rules in the spec.  It would be nice to have them for all predicates.  If your still looking for project ideas for that event you're organizing I meant to add that to the list. :)